mysql-test/suite/galera/r/galera_flush_local.result

```diff
@@ -3,6 +3,7 @@ Table	Op	Msg_type	Msg_text
 test.t0	optimize	Warning	Truncated incorrect sort_buffer_size value: '0'
 test.t0	optimize	Warning	Truncated incorrect myisam_repair_threads value: '0'
 test.t0	optimize	Error	Table 'test.t0' doesn't exist
+test.t0	optimize	Error	Table 'test.t0' doesn't exist
```

Comment:
Yes, it is expected. I mean this is because right now we are adding parent tables as writeset keys (wsrep_append_fk_parent_table() ). To do this, we are opening the given table and traversing through its FK's. As the table is not present (this is the test) the error is pushed to the stack. Next, the original flow continues and... well... the table does not exist.
I was thinking about suppressing the 1st error, but I think it is better to have it there for diagnostic purposes.

Please also note that this behavior comes from upstream (Codership)